### PR TITLE
add Device:otaModel()

### PR DESCRIPTION
**Pull request by pazos**
Devices with a single target might want to specify it in `Device.ota_model`
Devices with multiple targets want to override the function or to specify `ota_model` variants for each target.

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/12014)

### CODE REVIEW

**Comment by pazos** [comment]
Does "sony-prstux" still receive OTA updates?

**Review by NiLuJe — APPROVED** [review]
LGTM ;).

**Comment by Frenzie** [comment]
> Does "sony-prstux" still receive OTA updates?

No, mainly to save some server space.

File: frontend/device/android/device.lua [hunk 1]
```diff
@@ -115,6 +115,21 @@ local Device = Generic:extend{
         end
     end,
 }
+function Device:otaModel()
```
Comment (Frenzie) [inline frontend/device/android/device.lua:118]:
```suggestion

function Device:otaModel()
```

**Comment by mergen3107** [comment]
Did this PR pass all checks?
How come it missed that `getZsyncFilename` function was not defined at all in `otamanager.lua`?

Currently there are three calls to it in that file.

**Comment by Frenzie** [comment]
See `https://github.com/lunarmodules/luacheck/issues/39`.

**Pull request merged**